### ISSUE TITLE
[Feature Request] Verbosity configuration

### ISSUE THREAD

**Issue by jvacek** [adamchainz/django-watchfiles#1]
👋 

Thanks for the package, looking promising!

I wouldn't mind if there were a settings variable for managing the verbosity level, it's a bit too much rn.

Thanks!

**Comment by adamchainz** [comment]
Good point - when I come back to finishing this off, I'll try align that with Django's verbosity setting.

How's the actual watching? Is it working well? How big is your project? Thanks

**Comment by jvacek** [comment]
So far so good! I've been using it on a smaller internal project that's only one app, and I've noticed nothing that's breaking my flow.

**Comment by adamchainz** [comment]
Brilliant

**Comment by q0w** [comment]
@adamchainz What do you think about rewriting `autoreload.run_with_reloader` instead of `autoreload.get_reloader` to get verbosity level and then pass `debug=True` or `debug=False` depending on the verbosity level (e.g. if verbosity >1 then debug=True)

**Comment by adamchainz** [comment]
that sounds feasible but I haven't been looking at the codebase in a while... would you mind making a PR?

**Comment by q0w** [comment]
@adamchainz done
